Connected System Template (CST) Examples/Google Drive (OAuth,  Sending Files and Multiple Operations)/src/main/resources/appian-plugin.xml

```diff
@@ -9,9 +9,14 @@
 
   <connected-system-template key="GoogleDriveSampleConnectedSystem" name="GoogleDriveSampleConnectedSystem"
                              class="com.mycorp.helloworld.templates.GoogleDriveSampleConnectedSystemTemplate">
-
+    <!--
+    In order for both of the Send File and Create Folder templates to be registered, 2 integration-template tags
+    need to be included. It's important to note that the keys have to be different.
```

Comment:
an integration-template tag for each must be included within the connected-system element